### PR TITLE
Added module for L4 Policy Set

### PR DESCRIPTION
**Pull request by chaitanyaavi**
##### SUMMARY

Added module for L4 Policy Set

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

network/avi/avi_l4policyset.py

##### ANSIBLE VERSION

```
ansible 2.6.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/vinay/git/avitest/local/lib/python2.7/site-packages/ansible
  executable location = /home/vinay/git/avitest/bin/ansible
  python version = 2.7.6 (default, Nov 23 2017, 15:49:48) [GCC 4.8.4]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

**Comment by grastogi23** [comment]
@chaitanyaavi  pl. update the module name and re-test with the ansible devel.

File: lib/ansible/modules/network/avi/avi_l4policyset.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+#
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: avi_l4policyset
+author: Gaurav Rastogi (grastogi@avinetworks.com)
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_l4policyset.py]: This isn't your name

File: lib/ansible/modules/network/avi/avi_l4policyset.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    state:
+        description:
+            - The state that should be applied on the entity.
+        default: present
+        choices: ["absent", "present"]
+    avi_api_update_method:
+        description:
+            - Default method for object update is HTTP PUT.
+            - Setting to patch will override that behavior to use HTTP PATCH.
+        version_added: "2.5"
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_l4policyset.py]: Remove this line

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged into `devel` will be released in Ansible 2.6.0

**Pull request merged**